app/src/main/kotlin/com/simplemobiletools/gallery/pro/dialogs/PickDirectoryDialog.kt

```diff
@@ -72,6 +79,48 @@ class PickDirectoryDialog(
         fetchDirectories(false)
     }
 
+    private fun configureFolderSearchView() = with(searchView) {
+        updateHintText(context.getString(R.string.search_folders))
+        searchEditText.imeOptions = EditorInfo.IME_ACTION_DONE
+
+        toggleHideOnScroll(!config.scrollHorizontally)
+        setupMenu()
+        onSearchClosedListener = {
+            searchEditText.clearFocus()
+            activity.hideKeyboard(searchEditText)
```

Comment:
these two is already implemented in `MySearchMenu`, but not working properly if the view is used in an alert dialog. 
the fix can be applied on `Simple-Common`